library/health_check.py

```diff
@@ -73,7 +75,7 @@ def is_zero(self):
     def __str__(self):
         amount_str = self.amount
         if self.amount:
-            amount_str = f"*{amount_str}*"
+            amount_str = f"**{amount_str}**"
```

Comment:
Looks like this still causes literal asterisks to appear in Slack. For now I think we have to settle for one asterisk and have it appear in Italic in HTML

![CleanShot 2023-10-23 at 13 32 07@2x](https://github.com/SACGF/variantgrid/assets/2100255/4e2e6f36-66a4-4bfe-9378-90456735c94b)